src/gei/Handlers/GenerateScriptCommandHandler.cs

```diff
@@ -481,11 +485,13 @@ private string GetGhesRepoOptions(string ghesApiUrl, string awsBucketName, bool
         return $"--ghes-api-url \"{ghesApiUrl}\"{(awsBucketName.HasValue() ? $" --aws-bucket-name \"{awsBucketName}\"" : "")}{(noSslVerify ? " --no-ssl-verify" : string.Empty)}";
     }
 
-    private string WaitForMigrationScript(string repoMigrationKey = null) => $"gh gei wait-for-migration --migration-id $RepoMigrations[\"{repoMigrationKey}\"]";
+    private string WaitForMigrationScript(string repoMigrationKey = null) {
+        return $"if ($RepoMigrations[\"{repoMigrationKey}\"]) {{ gh gei wait-for-migration --migration-id $RepoMigrations[\"{repoMigrationKey}\"]\nif ($lastexitcode -eq 0) {{ $Succeeded++ }} else {{ $global:failed++ }} }}";
+    }
 
     private string DownloadMigrationLogScript(string githubTargetOrg, string targetRepo)
     {
-        return $"gh gei download-logs --github-target-org \"{githubTargetOrg}\" --target-repo \"{targetRepo}\"";
+        return $"if ($RepoMigrations[\"{targetRepo}\"]) {{ gh gei download-logs --github-target-org \"{githubTargetOrg}\" --target-repo \"{targetRepo}\" }}";
```

Comment:
We only want to download the logs if a migration ID was recorded for the repo. If no migration started, we can skip it.